kyo-prelude/shared/src/main/scala/kyo2/Result.scala

```diff
@@ -88,6 +89,23 @@ object Result:
         def isPanic: Boolean =
             self.isInstanceOf[Panic]
 
+        def value: Maybe[A] =
```

Comment:
New convenience methods. It's great that we can provide this kind of API without worrying about allocations!

Reply:
Can these be supported by `unapply`? Or is there a limitation?

I don't see an `unapply` for `Panic`

Reply:
Yes, it's still possible to use the extractors. `Panic` is already has an extractor since it's a `case class`. We have a custom extractor for `Error` to help the compiler infer the `E` type correctly but `Panic` doesn't need it since it extracts a `Throwable` instead of a parametrized value like in `Error`.